.releaserc

```diff
@@ -0,0 +1,146 @@
+{
+  "branches": [
+    "master",
+    {
+      "name": "next",
+      "prerelease": true
+    }
+  ],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "angular",
+        "releaseRules": [
+          {
+            "type": "feat",
+            "release": "minor"
+          },
+          {
+            "type": "style",
+            "release": "patch"
+          },
+          {
+            "type": "refactor",
+            "release": "patch"
+          },
+          {
+            "scope": "skip",
+            "release": false
+          }
+        ],
+        "parserOpts": {
+          "noteKeywords": [
+            "BREAKING CHANGE",
+            "BREAKING CHANGES",
+            "BREAKING"
+          ]
+        }
+      }
+    ],
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "conventionalcommits",
+        "presetConfig": {
+          "types": [
+            {
+              "type": "feat",
+              "section": "✨ New & Improved",
+              "hidden": false
+            },
+            {
+              "type": "refactor",
+              "section": "✨ New & Improved",
+              "hidden": false
+            },
+            {
+              "type": "style",
+              "section": "💄 Style Refinements",
+              "hidden": false
+            },
+            {
+              "type": "fix",
+              "section": "🛠 Fixes & Updates",
+              "hidden": false
+            },
+            {
+              "type": "perf",
+              "section": "🛠 Fixes & Updates",
+              "hidden": false
+            },
+            {
+              "type": "chore",
+              "section": "🛠 Fixes & Updates",
+              "hidden": false
+            },
+            {
+              "type": "docs",
+              "section": "📘 Tests & Docs",
+              "hidden": false
+            },
+            {
+              "type": "test",
+              "section": "📘 Tests & Docs",
+              "hidden": true
+            },
+            {
+              "type": "build",
+              "hidden": true
+            },
+            {
+              "type": "ci",
+              "hidden": true
+            }
+          ]
+        },
+        "parserOpts": {
+          "noteKeywords": [
+            "BREAKING CHANGE",
+            "BREAKING CHANGES",
+            "BREAKING"
+          ]
+        },
+        "writerOpts": {
+          "commitsSort": [
+            "subject",
+            "scope"
+          ],
+          "headerPartial": "## Version {{version}}"
+        }
+      }
+    ],
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogTitle": "Changelog\n==="
+      }
+    ],
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": true,
+        "tarballDir": "dist"
```

Comment:
idk what to change this to since we're using Lerna.

Reply:
Responded [here](https://github.com/readmeio/api-explorer/pull/828#discussion_r454000133).